src/index.html

```diff
@@ -9,6 +9,56 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="moyo_logo">
+        <a href="#home">
+          <img
+            src="images/logo.png"
+            alt="moyo_logo"
+          >
+        </a>
+      </div>
+      <nav class="nav__list">
+        <ul>
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+            href="#laptops & computers"
+            class="nav__link"
+            data-qa="hover"
+            >
+            Laptops & Computers
+            </a>
```

Comment:
```suggestion
            <a
              href="#laptops & computers"
              class="nav__link"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```
read the [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md):
![image](https://github.com/mate-academy/layout_moyo-header/assets/86923089/9817eda8-d4e6-442a-a6af-fa2e40025397)